app/application.py

```diff
@@ -74,6 +79,14 @@ def on_application_start(self):
         # Initialize main window
         self.ui = self.main_widget()
 
+        if self.account_balance_zero():
```

Comment:
This is executed just after `self.main_widget()` has started the managed node process. The node might still be loading and not be able to answer RPC-requests.